paying_for_college/disclosures/urls.py

```diff
@@ -10,10 +10,13 @@
         name='pfc-feedback'),
 
     url(r'^technote/$',
-        TemplateView.as_view(template_name="technote.html"),
+        BaseTemplateView.as_view(template_name="technote.html"),
         name='pfc-technote'),
 
-    url(r'^api/search-schools.json', school_search_api),
+    url(r'^api/search-schools.json',
+        school_search_api,
+        name='school_search'),
+
```

Comment:
I realize this isn't really a change, but I'm going to comment on it anyway: is there a reason the endpoint ends with `.json`?